source/Public/New-GuestConfigurationPolicy.ps1

```diff
@@ -96,6 +115,11 @@
         A hashtable of the tags that should be on machines to apply this policy on.
         If this is specified, the created policy will only be applied to machines with all the specified tags.
 
+
+    .PARAMETER ExcludeArcMachines
+        This parameter needs to be specified if the New-GuestConfigurationPolicy is using a User Assigned Identity.
```

Comment:
This needs to mention the ManagedIdentityResourceId parameter
You can use this without identities too, but this makes it sound like it's only for identity

Reply:
We don't want to disclose this immediately as a feature as per the PMs. We will give the functionality but not document it just yet.